### ISSUE TITLE
Update content and link in Change Address page

### ISSUE THREAD

**Issue by tressaellen** [department-of-veterans-affairs/va.gov-team#8420]
### Background

[In address validation user testing](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/change-address/address-validation/user%20testing/Research%20Summary.md#showing-a-sign-in-link-on-the-change-your-address-content-page-is-confusing-when-a-user-is-already-signed-in), we noticed the "Change your address" content page uses subway map directions to tell you how to change your address. This means we show "Step one: Sign in to VA.gov" even when people are already signed in. This caused some confusion for participants who thought they had to sign in again. 

Also with the new profile we need to update the content and links to be accurate so users don't get lost.

## Additional Information

Epic #8395 
Related to ticket #5524 and #3760

![Screen Shot 2020-04-24 at 9.36.48 AM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/7e0f4dbe-496a-475b-95cf-076ee7751a19)

### Design & Content

I recommend changing the subway map to a dynamic modal. We will need the following 3 states:

1. Sign in to VA.gov
2. Verify your identity
3. Go to the VA.gov profile

![Change address content page](https://user-images.githubusercontent.com/34068740/69677988-41887700-1072-11ea-8c96-4c6160e01172.png)

## Tasks

- [ ] Tressa update designs to reflect all dynamic modal states
- [ ] Tressa work with Peggy for updated content
- [ ] Update 'Go to your va.gov profile' link

## Acceptance criteria

- [ ] Design/content have been updated.
- [ ] This section has been built and reviewed by Tressa.

**Comment by tressaellen** [comment]
@mshea0606 please review

**Comment by mshea0606** [comment]
@tressaellen updated

**Comment by Samara-Strauss** [comment]
How would you feel about rolling ticket #3760 into this one? I wonder if that would actually simplify this task in some ways. You'd just have to make sure all states of the dynamic widget are accounted for (should be signed out, LOA1, LOA3).

**Comment by tressaellen** [comment]
@Samara-Strauss I think this is a great idea - only one of the users in testing used the subway map to get to the profile page and was confused about it as well. It would simplify things a lot to have dynamic widgets.

**Comment by Samara-Strauss** [comment]
@tressaellen YAY! Awesome. This will be a great update to this page!

**Comment by tressaellen** [comment]
Hi @peggygannon, I would love your input on this page and the widget content. The original page is [here](https://www.va.gov/change-address/).  This 'subway map' version caused some confusion for participants because it would show this content regardless of if a user was signed in or not. We're moving to widgets to reflect whatever step the user is in. Also since we are now combining profile and account I think some of the broader content could be streamlined a bit. 

I suggest moving the 'How do I change my address in my VA.gov profile?' up to the top. Any thoughts on that?

I'm specifically looking for any content changes/updates for:

- the blurb right after 'How do I change my address in my VA.gov profile' (You can update your home and mailing addresses...)
- content for the 'Go to my va.gov profile' - I think it should take them directly to the personal and content information section, not to the profile
- Any other updates in the rest of the original page would be great.

Thank you!

**Comment by peggygannon** [comment]
Hello, @tressaellen  I have a few followup questions: 

1. Looking at above comments it sounds like the subway map is going away  and being replaced by the sign in widget? (3 different states) . Can you confirm if that is correct? 

2. If a user is already signed in and LOA3, then will they see the widget at all? 

3. If the subway map is going away, then do you still need content for ~"'Go to my va.gov profile'  link?  I think it should take them directly to the personal and content information section, not to the profile~

**Comment by Samara-Strauss** [comment]
Also, @tressaellen, it would be helpful to see mockups of the different states of the page. This should generally be our practice before pulling Peggy in for copy help.

**Comment by tressaellen** [comment]
@Samara-Strauss and @peggygannon , oh my goodness, I'm so sorry! I could've sworn I added the screenshots with that message, that definitely would have made things easier! Here are the screenshots: 

If a user needs to sign-in:
![Screen Shot 2020-06-25 at 8.49.19 AM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/51cd12e1-646c-4453-a2ba-41c7e03f91af)  

Verify your identity:
![Screen Shot 2020-06-25 at 8.49.24 AM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/fe7a5881-8bdc-41b8-84ad-b009de892a03)


Go to my profile:
![Screen Shot 2020-06-25 at 8.49.31 AM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/9345778a-1269-403e-a195-8614f6892a4b)

And here's answers to your questions - even though I think the screenshots probably answer some of them. 

1. Yes that is correct, the subway map is being replaced by widgets.

2/3. Yes the LOA3 user would see the 'Go to your profile' widget. The landing page of the profile is the Personal and contact information section in profile 2.0, so if they click on the 'Go to your profile' button they should go directly there. I'm definitely open to any language changes that might make that clearer.

Sorry again for missing those screenshots!

**Comment by Samara-Strauss** [comment]
Questions:

- Is there anything else on that page that needs to be updated? The answer to this may be no, but just confirming.
- Can you confirm which colors we should be using for each alert/CTA? I know we sometimes use the blue box/blue CTA but I personally don't know what the guidance is on those at this point. You may have already confirmed this for each state of the alert, so just checking.

**Comment by tressaellen** [comment]
@Samara-Strauss 

- The only other part of the page that might need need some updating is if the contents of this list have changed at all and need to be updated. If they haven't then I think the rest of the current content works well. 
![Screen Shot 2020-06-25 at 11.53.14 AM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/e896e8ed-cfe6-4590-b818-a8bae0a04fae) 
- I talked to platform about this, and they said that the green was the correct color for the CTA buttons since they are the primary action for the page

**Comment by Samara-Strauss** [comment]
Awesome, thanks for this! Heard loud and clear on the button/CTA colors (I'm wondering now if we are handling that correctly on all our content pages, but that's a problem for another day). Also, that content blurb is still correct AFAIK.

**Comment by tressaellen** [comment]
Okay great! I had the same thought about the CTA buttons, it'd be interesting to take a look at those.

**Comment by peggygannon** [comment]
Per our convo today with PW content team, that team will handle the content updates to change-address page. 

@DanielleSoCompany @lunascoop @bethpottsVADEPO 


to summarize what is happening: 

1. subway map is being being replaced by widgets. Depending on user's sign in status (signed out, LOA1, or LOA3), they will see the widget that applies to their sign in status.  (see the screenshots above from @tressaellen)

2. The LOA3 user would see the 'Go to your profile' widget. The landing page of the profile is the Personal and contact information section in profile 2.0, so if they click on the 'Go to your profile' button they should go directly there. I'm definitely open to any language changes that might make that clearer.

3. need to work with PW devs to see if these widgets exist and confirm what elements of the widgets are editable. 


Here is the draft intro content that would appear above the widgets..

**How do I change my address in my VA.gov profile?**

You can update your address in your profile's personal and contact information section. 

You need to be signed in to your VA.gov account and your identity verified before you can update your address. Once you’re signed in and verified, go to your VA.gov profile by clicking on the icon with your name in the top right corner of any VA.gov page.

**Comment by tressaellen** [comment]
@peggygannon I think that intro content works really well - I'll drop in some screenshots shortly. 

There are a couple other widgets that are _very similar_ on the [change your direct deposit page](va.gov/change-direct-deposit/).  They also change depending on the users logged in status.  If a user is not logged in then I think we could just change the title to 'Go to your VA.gov profile to change your address'
![Screen Shot 2020-06-26 at 2.34.34 PM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/33946047-0204-4e7f-b1c5-1e83ac31ca8e)

We would need to update the title and the content for the widget if the user is logged in. 
![Screen Shot 2020-06-26 at 2.30.14 PM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/06482cfc-5bce-43ae-bd4e-77c75f36eae4)

For verifying an identity - that widget exists on the profile already in account security, but I'm not sure how that translates to including it in this content page.

**Comment by tressaellen** [comment]
And here is the screenshot with the updated intro content. 

![Screen Shot 2020-06-26 at 2.38.37 PM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/98fd37e8-5dbe-4aaf-a76b-896fb1883ce5)

**Comment by DanielleThierryUSDSVA** [comment]
Hi @tressaellen. Just checking in to let you know I'll have content for you by EOD (likely late) today. Thanks!

**Comment by tressaellen** [comment]
Sounds good - thank you @DanielleThierryUSDSVA

**Comment by DanielleThierryUSDSVA** [comment]
Hi @tressaellen. Recommended content below. I'm confirming with our engineers about the widgets and editability tomorrow morning, and will post update then.

## How do I change my address in my VA.gov profile?

You can change your mailing and home addresses in your profile's **Personal and contact information** section. To do this, you need to be signed in to VA.gov with a verified account.

### [sign in to va.gov]

**Please sign in to change your address**

Try signing in with your **DS Logon**, **My HealtheVet**, or **ID.me** account. If you don’t have any of these accounts, you can create one now.

[button] **Sign in or create account**

### [verify your identity]

**Please verify your identity to change your address**

We need to make sure you’re you—and not someone pretending to be you—before we can give you access to your personal and health-related information. This helps us keep your information safe and prevent fraud and identity theft.

**This one-time process takes about 5 to 10 minutes.**

[button] **Verify your identity**

#### [go to the va.gov profile]

**Go to your VA.gov profile to change your address**

You’ll find your home and mailing addresses in your profile's **Personal and contact information** section.

[button] **Go to your VA.gov profile**
.

**Comment by Samara-Strauss** [comment]
This content looks great to me!! Thank you, Danielle!

**Comment by tressaellen** [comment]
Thanks @DanielleThierryUSDSVA ! This all looks great, here are screenshots for the design.  The only question I have is with the [verify your identity] widget, you've changed the button from 'Verify **my** identity' to 'Verify **your** identity' - do you suggest that we change that on the LOA1 screen as well? (That screen has the same alert)

[sign in to VA.gov]

![Screen Shot 2020-07-01 at 8.52.56 AM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/3a74a2fa-d73d-421d-bde3-457aa164a289)

[verify your identity]

![Screen Shot 2020-07-01 at 8.54.26 AM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/156fbfbb-3101-4f1a-a027-31d9522fff98)

[go to the va.gov profile]

![Screen Shot 2020-07-01 at 8.56.37 AM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/935f388f-4061-468f-8cc8-91f56f40a628)

**Comment by DanielleThierryUSDSVA** [comment]
Thanks, @Samara-Strauss @tressaellen! Yes, if we can change that across the board, that would be great. Thanks!

**Comment by tressaellen** [comment]
Okay - will do! Thank you!

**Comment by DanielleThierryUSDSVA** [comment]
Hi @tressaellen. Also confirmed with our engineers that we can customize the alerts as needed. Will create a new issue on our end for the changes. Thanks!

**Comment by tressaellen** [comment]
Sounds great - let me know if you need anything else from me.

**Comment by Samara-Strauss** [comment]
Implementation tickets are #10853 and #10857.
